### DISCUSSION TITLE
Drill down menu

### DISCUSSION THREAD

**Discussion by romansndlr** [adobe/react-spectrum#7450] [General]
What would be the best way to create an experience similar to [this](https://get.foundation/sites/docs/drilldown-menu.html#:~:text=Drilldowns%20use%20the%20standard%20Menu,ul%3E%20that%20it's%20next%20to.)?

I know we have sub menus but this is a little different and I'm not sure what would work best. 

Also, my use case is a field inside of a form so ideally I would want a select and not a menu

**Comment by reidbarber** [comment]
We implemented this on mobile devices for the [submenu](https://react-spectrum.adobe.com/react-spectrum/Menu.html#submenus) in React Spectrum, however I'm not sure there is currently a good solution for doing this as a form field.
